app/models/notification.rb

```diff
@@ -48,10 +48,6 @@ class Notification < ApplicationRecord
 
   cache_associated :from_account, status: STATUS_INCLUDES, mention: [status: STATUS_INCLUDES], favourite: [:account, status: STATUS_INCLUDES], follow: :account
 
-  def activity(eager_loaded = true)
```

Comment:
The method signature changed here, and there is no longer an `eager_loaded` argument passed in.

Removing this did not break any specs ... but my guess is there was just no coverage. Worth investigating how this was used (probably avoid queries?) and make sure it's not a regression.